.github/workflows/test-chart-version-nightly-template.yaml

```diff
@@ -0,0 +1,70 @@
+# Basic test for maintenance Camunda 8 Helm charts.
+name: "Test - Chart Version Nightly - Template"
+
+on:
+  workflow_call:
+    inputs:
+      chart-version:
+        description: Chart version to test
+        required: true
+        type: string
+      scenarios:
+        description: Scenarios
+        required: true
+        type: string
+      type:
+        description: Type
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ inputs.chart-version }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  validation:
+    name: Camunda ${{ inputs.chart-version }} - Validation
+    uses: ./.github/workflows/chart-validate-template.yaml
+    with:
+      identifier: "${{ inputs.type }}-vald-${{ inputs.chart-version }}"
+      camunda-helm-dir: "camunda-platform-${{ inputs.chart-version }}"
+      camunda-helm-git-ref: "${{ github.event.pull_request.head.sha }}"
+  format-scenarios:
```

Comment:
I'd just call it `init`, following the same pattern we have in other workflows.

Reply:
done